log/log.go

```diff
@@ -15,6 +18,10 @@ type Logger interface {
 	Log(keyvals ...interface{}) error
 }
 
+// ErrMissingValue may be appended to keyval slices with an odd length to
```

Comment:
rephrase suggestion: ErrMissingValue is appended to keyvals slices with odd length to substitute the missing value